src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -112,6 +112,7 @@
 		mutableAudioOrVideoContainers.put(FormatConfiguration.THREEGPP2, new AudioVariantInfo(new THREEG2A(), FormatConfiguration.THREEGA));
 		// XXX WEBM Audio is NOT MKA, but it will have to stay this way until WEBM Audio is implemented.
 		mutableAudioOrVideoContainers.put(FormatConfiguration.WEBM, new AudioVariantInfo(new MKA(), FormatConfiguration.WEBA));
+		mutableAudioOrVideoContainers.put(FormatConfiguration.WMV, new AudioVariantInfo(new WMA(), FormatConfiguration.WMA));
```

Comment:
@Sami32 As with all the "audio or video containers" UMS parse it as video first, and then change it to audio later if the content turns out to be only a single audio track. WMV and WMA aren't "formats", Windows Media is. Windows Media can be either audio/video or just audio (or probably just video as well). The point is that as long as it *is* a Windows Media (WM) format and it only contains one audio track, we should treat it as audio media (and thus WMA). The file extension isn't really relevant in that context.

If, like you say, ASF is parsed as WMV, that is the real problem - as that's simply not always the case. I have very few ASF files and MediaInfo claims that all of them is WM. I don't know what's really the case, a non-WM ASF file would be needed to test where it goes wrong.